rust/pact_matching/src/models/matchingrules.rs

```diff
@@ -214,6 +214,8 @@ impl <T: Display> DisplayForMismatch for &[T] {
 /// Set of all matching rules
 #[derive(Serialize, Deserialize, Debug, Clone, Eq)]
 pub enum MatchingRule {
+  /// Dummy rule to allow all values passed in via the FFI to be JSON integration values, even primitives
+  Dummy,
```

Comment:
There must be a better way. I'll have a look into it.

Reply:
I figured you might say that. Thanks, any ideas would be great.